force-app/main/default/classes/InterviewReminder.cls

```
@@ -0,0 +1,161 @@
+                    System.debug(e.getMessage());
+                } 
+            }
+        }
+        return emails;
+    }
+    //Batchable Step 7: Send emails
+    public void sendInterviewReminderEmails(List<Messaging.SingleEmailMessage> emails){
+        if(!emails.isEmpty()){
+            List<Messaging.SendEmailResult> emailResults =  Messaging.sendEmail(emails, FALSE);
+            for(Messaging.SendEmailResult emailResult : emailResults){
+                System.debug(emailResult.isSuccess());
+            }
+        }
+    }
+    //Batchable Step 6: Execute steps 2 to 7
+    public void execute(Database.BatchableContext bc, List<Event> scope){
+        try{
+            if(!scope.isEmpty()){
+                //Step 2
+                System.debug('Scope is not empty');
+                Set<Id> jobApplicationIds = this.getJobApplicationIds(scope); 
+                System.debug('Job Application Ids: ' + jobApplicationIds);
+                //Step 3
+                List<Job_Application__c> jobsList = this.getJobApplicationList(jobApplicationIds);
+                System.debug('jobsList: ' + jobsList );
+                //Step 4
+                Map<Id, String> jobApplicationToAccountNameMap = this.mapJobApplicationsToAccount(jobsList); 
+                System.debug(jobApplicationToAccountNameMap);
+                //Step 5
+                Map<Id, String> jobApplicationToJobApplicationNameMap = this.mapJobApplicationIdToJobApplicationName(jobsList);
+                System.debug(jobApplicationToJobApplicationNameMap);
+                //Step 6
+                List<Messaging.SingleEmailMessage> emailList = this.prepareEmails(scope, jobApplicationToAccountNameMap, jobApplicationToJobApplicationNameMap); 
+                System.debug(emailList);
+                //Step 7
+                this.sendInterviewReminderEmails(emailList);
+            }
+        }catch (Exception e){
+            System.debug(e.getMessage());
+        }
```

Dont try/catch if you dont do anything in the catch. This will look like a success in the batch apex logs and you wont know there are errors. If you don't something to do in handling the error, let it fail.